pkgs/applications/version-management/git-and-tools/git-branchless/default.nix

```
@@ -48,6 +48,14 @@ rustPlatform.buildRustPackage rec {
     "--skip=test_next_ambiguous_interactive"
   ];
 
+  patches = [
+    (fetchpatch {
```

I would be ok, with pulling patches. It sometimes can take a while depending on the project before a release
appears:

```suggestion
    # fixed in next release
    (fetchpatch {
```

---

Added the comment and rebased the PR.